content/_data/plugins.json

```
@@ -342,6 +342,13 @@
           "link": "https://github.com/nils-hoyer/cypress-fail-on-console-error",
           "keywords": ["console", "error", "fail"],
           "badge": "community"
+        },
+        {
+          "name": "Cypress Chrome Recorder Extension",
+          "description": "Official Cypress extension for Chrome DevTools that allows you to export tests directly from the Recorder panel.",
```

Feels worth specifying that you're really getting the good stuff here as Cypress users. 

```suggestion
          "description": "Official Cypress extension for Chrome DevTools that allows you to export Cypress tests directly from the Recorder panel.",
```